webapp-django/crashstats/base/views.py

```diff
@@ -3,6 +3,16 @@
 from django.shortcuts import render
 
 
+class HttpResponseBadRequest(http.HttpResponseBadRequest):
+    """Override of the Django HttpResponseBadRequest that makes sure
+    to set the content_type to "text/plain" if it hasn't already been
+    set."""
+    def __init__(self, *args, **kwargs):
+        if 'content_type' not in kwargs:
+            kwargs['content_type'] = 'text/plain; charset=UTF-8'
```

Comment:
The assertion would improve our confidence that we didn't miss something and that all the places this is used have the appropriate content-type which seems to vary depending on the situation in ways that don't seem easy to skim (at least not for me).

You're right that it probably results in more typing and it'll take longer to do.